notebooks/article_1.py

```diff
@@ -0,0 +1,1201 @@
+# - - The irrelevant features
+# - - The missing values
+# - - The negative values where it shouldn't be so.
+#
+# - The **Data transformation** step modifies data to re-organize them into a format appropriate for performing analysis. These are the common transformatio that we will perform:
+# - - **Normalization** and **standardization**: These techniques adjust data to a common scale, making it easier to compare them.
+# - - **Encoding categorical data**: Transforming categorical variables into numerical formats.
+
+# ### Handle Irrelevant Features
+
+# Irrelevant feature refers to a data attribute or variable that does not provide valuable or meaningful information for the task at hand. Here are some reasons why removing them is beneficial:
+#
+# - Eliminating irrelevant variables simplifies the analysis process.
+# - It ensures focus on meaningful factors by reducing clutter and improving clarity in data exploration.
+#
+# To accomplish this, we will implement the following method.
+
+# In[6]:
+
+
+data_df = data_df.drop([
+    'Unnamed: 0',   #Irrelevant data
+    'latitude',     #Irrelevant data
+    'longitude',    #Irrelevant data
+    'zip_code',     #Irrelevant data
+    'id',           #Irrelevant data
+    'Unnamed: 6',   #Irrelevant data
+    'name',         #Irrelevant data
+    'labels',       #Same data as status
+    'state_code.1', #Same data as state_code
+    'is_CA',        #Binary column, not needed
+    'is_NY',        #Binary column, not needed
+    'is_MA',        #Binary column, not needed
+    'is_TX',        #Binary column, not needed
+    'is_otherstate',#Binary column, not needed
+    'is_software',  #Binary column, not needed
+    'is_web',       #Binary column, not needed
+    'is_mobile',    #Binary column, not needed
+    'is_enterprise',#Binary column, not needed
+    'is_advertising',#Binary column, not needed
+    'is_gamesvideo',#Binary column, not needed
+    'is_ecommerce', #Binary column, not needed
+    'is_biotech',   #Binary column, not needed
+    'is_consulting',#Binary column, not needed
+    'is_othercategory',#Binary column, not needed
+    'object_id'],   #Irrelevant data
+    axis=1,
+).copy()
+data_df.head(2)
+
+
+#
+# After discarding irrelevant variables, we've simplified the DataFrame. From 50 columns initially, it now contains 24 columns. Let's look at the data.
+
+# In[7]:
+
+
+data_df.info()
+
+
+# A preliminary review allows us to understand the nature of the data, especially focusing on identifying key data types that will help us to make decisions.
+#
+# Before starting to do any data cleaning. Let's begin with an overview of some categorical variables such as `category_code` and `status` that will help us to gain valuable insights.
+#
+# To achieve this, we will utilize **bar chart** visualizations, known for their effectiveness in displaying data
```

Comment:
"Let's look at the data." is too simplistic.

Reply:
done